lib/datadog/tracing/contrib/delayed_job/ext.rb

```diff
@@ -21,6 +21,8 @@ module Ext
           TAG_OPERATION_ENQUEUE = 'enqueue'.freeze
           TAG_OPERATION_JOB = 'job'.freeze
           TAG_OPERATION_RESERVE_JOB = 'reserve_job'.freeze
+
+          TAG_MESSAGING_SYSTEM = 'delayed_job'.freeze
```

Comment:
Sounds good! I'll use the `TAG_COMPONENT` value for integrations where we are not using the message broker itself like kafka, amazonsqs, rabbitmq, etc.

Also I was wondering if there's a good place to put these whitelisted values as mentioned above, that can appear in several integrations, instead of repeating it in the `ext.rb` file for each one?